README.md

```diff
@@ -31,20 +33,18 @@ If you do not have Node.js installed, you can download it from the official [Nod
 ## Installation
 
 1. **Clone the repository** (or download the ZIP)
+
    ```bash
    git clone https://github.com/klever-io/klever-docs.git
    ```
 
 2. **Navigate to the project folder**
+
    ```bash
    cd klever-docs
    ```
 
 3. **Install dependencies**
-   ```bash
-   npm install
-   ```
-   or
```

Comment:
readme says to use `npm` or `yarn` even do we actually use `pnpm` if we are making changes here, lets do instructions to `pnpm`